docs/source/tutorial/schema.mdx

```diff
@@ -187,13 +187,13 @@ By default, Apollo Server supports [GraphQL Playground](https://www.apollographq
 The GraphQL Playground provides the ability to introspect your schema. **Introspection** is a technique used to provide detailed information about a graph's schema. To see this in action, check out the right hand side of GraphQL Playground and click on the `schema` button.
 
 <div style="text-align:center">
-  <img src="../images/schematab.png" alt="Schema button">
```

Comment:
We could probably undo all of the changes to this file and turn it back into a regular old MD file to reduce the footprint of this PR. This should be straightforward to do using git, and I'd be happy to help do that if you agree with the change!

Reply:
I agree, yeah @trevorblades would you mind reverting it for me? Thanks!

Reply:
Done! 🎉